src/index.tsx

```diff
@@ -2,12 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App';
 
-
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
\ No newline at end of file
+
+root.render(<App />);
```

Comment:
I removed strict mode for performance reasons. I'm not sure what this gives us, to be honest. It's been a while since I looked at strict mode. But it was rendering your app twice, which was making your calls all fire twice as well. With how many calls you were firing anyway, this was getting expensive quickly, so I just removed it.

Reply:
Thanks J.  Good catch and call.